### ISSUE TITLE
Plugin not reflecting the right number of sampler steps

### ISSUE THREAD

**Issue by rexelbartolome** [DrCyanide/cyanic-sd-krita#23]
Not entirely sure what triggers it but I often switch tabs and when I changed I2I to Settings to SimplifyUI to I2I again, the number of steps don't update. It still shows "20" even tho when I press generate, the instance shows that it's only using 10 steps etc.

**Comment by DrCyanide** [comment]
Settings on Img2Img, Txt2Img, and Inpainting are only saved when you click "Generate". This is to prevent accidental changes (which can happen if you use the scroll wheel over a number or drop down select field) from being saved. If I could disable the scroll wheel changing those values then I'd implement auto-saving on them, but I've had too many instances where I accidentally change them when trying to scroll past.

If you're trying to change default values in the Simplify UI tab, you have to scroll to the bottom of the page and click the "Save" button. 

In other words, this is working as intended right now.
